Preferences.sublime-settings

```diff
@@ -40,6 +40,16 @@
 	// Adds <Tab> after list items instead of a single <space>.
 	"mde.list_align_text": false,
 
+	// MarkdownEditing (References):
+	// The sorting method used by the Organize References command.
+	// Should be one of
+	//
+	// 'marker_order': List in order of appearance in document
+	// 'alphabetical': Alphabetical based on reference name, sorting numerals lexagraphically
+	// 'numeric': Alphabetical based on reference name, sorting numeral chunks numerically
+	"mde.ref_organize_sort": "marker_order",
```

Comment:
I know those are called `marker` all over the code, but is it really the correct wording? Does it make sense to replace `marker_order` by `reference_order`?

Reply:
I can see a case for changing the name to `reference_order` here if you want.

Reply:
I'd vote for renaming.